### PR TITLE
feat: add op-scim-bridge chart

### PR DESCRIPTION
**Pull request by eddycharly**
This PR adds op-scim-bridge chart.

### CODE REVIEW

**Comment by eddycharly** [comment]
Ping @RiceBowlJr i can't put you in reviewers ?

File: charts/op-scim-bridge/values.yaml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+# -- Pod affinity for SCIM pods
+affinity: {}
+
+# -- Storage class for the SCIM PVC
+storageClassName: do-block-storage
+
+# -- Redis configuration, check [the upstream configuration](https://github.com/bitnami/charts/blob/master/bitnami/redis)
+redis:
+  enabled: true
+  image:
```
Comment (eddycharly) [inline charts/op-scim-bridge/values.yaml:97]: Why do we need to set the image explicitely ?

**Review by RiceBowlJr — APPROVED** [review]

File: charts/op-scim-bridge/values.yaml (hunk 1 above)
Comment (eddycharly) [reply charts/op-scim-bridge/values.yaml:97]: ping @RiceBowlJr

Comment (RiceBowlJr) [reply charts/op-scim-bridge/values.yaml:97]: Nope that's the default I think, don't remember why I put this here

**Pull request merged**